src/app/pages/assistant/assistant.component.scss

```
@@ -0,0 +1,365 @@
+  &.bot-message {
+    justify-content: flex-start;
+  }
+}
+
+.message-bubble {
+  max-width: 75%;
+  border-radius: 16px;
+  padding: 12px 18px;
+  font-size: 14px;
+  line-height: 1.7;
+  word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)